docs/source/monitors.rst

```diff
@@ -32,8 +32,8 @@ number of items were extracted and fails if it is less than the expected thresho
                 msg='Extracted less than {} items'.format(minimum_threshold)
             )
 
-A `Monitor` instance has the following properties that can be used to help you
-implement your monitors:
+A `Monitor` instance defines a monitor that includes your monitoring logic and
```

Comment:
And this would be better now:

```suggestion
A :class:`~spidermon.core.monitors.Monitor` instance defines a monitor that includes your monitoring logic and
```

Reply:
Good catch, implemented.